### PR TITLE
Test format data

### PR DESCRIPTION
**Pull request by monthol8th**
create changeDataTableToArray that change table of data into object

### CODE REVIEW

File: src/utilities/format-data.js [hunk 1]
```diff
@@ -12,3 +12,34 @@ export const changeInsideTagToArray = (body, targetTag, subTag) => (
   x(body, targetTag, [[subTag]])
 );
 
+export function changeDataTableToArray(html){
```
Comment (komcal) [inline src/utilities/format-data.js]: change to es6 syntax
``` javascript
export const changeDataTableToArray = (html) => {
```

**Comment by monthol8th** [comment]
Hi

File: src/utilities/format-data.js [hunk 2]
```diff
@@ -12,3 +12,34 @@ export const changeInsideTagToArray = (body, targetTag, subTag) => (
...
+  let data = [];
+  x(html, 'tr', [['td']])((err, d) => {
+    data = d;
+  });
+  const year = data[0][0].replace('ปี ', '');
+  data.forEach(e => e.reverse());
+  data.shift();
```
Comment (komcal) [inline src/utilities/format-data.js]: Not sure if merge line to 
```javascript
data.forEach(......).shift();
```
It still work?

Comment (monthol8th) [inline src/utilities/format-data.js]: It's can work but I think it's harder to understand what is the propose of .
By the way, I will change it.

**Pull request merged**